.github/workflows/build-test.yaml

```diff
@@ -23,6 +23,13 @@ jobs:
         with:
           name: conda_artifacts
           path: ${{ env.OUTPUT_DIR }}
+  docs-build:
+    if: github.ref_type == 'branch'
+    needs: conda
+    runs-on: ubuntu-latest
+    container: rapidsai/ci-conda:latest
+    steps:
+      - run: ./ci/build_docs.sh
```

Comment:
you'll need to use [actions/checkout](https://github.com/actions/checkout) first